pkgs/by-name/ge/gensort/package.nix

```
@@ -0,0 +1,57 @@
+{
+  fetchurl,
+  lib,
+  pkgs,
+  stdenvNoCC,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  name = "gensort";
+  version = "1.5";
+
+  src = fetchurl {
+    url = "https://www.ordinal.com/try.cgi/gensort-${version}.tar.gz";
+    sha256 = "sha256-f3VzeD2CmM7z3Uqh24IlyRTeGgz+0oOWXqILaYOKZ60=";
+  };
+
+  phases = [
+    "unpackPhase"
+    "installPhase"
+  ];
+
+  unpackPhase = ''
+    runHook preUnpack
+    tar xvf $src
+    runHook postUnpack
+  '';
+
+  nativeBuildInputs = [
+    pkgs.gcc
+    pkgs.zlib
+  ];
```

```suggestion
  ];
  
  env.NIX_CFLAGS_COMPILE = "-Wno-error=format-security";
```
No need for the other flags